zoo/src/main/scala/com/intel/analytics/zoo/serving/http/FrontEndApp.scala

```
@@ -57,6 +57,41 @@ object FrontEndApp extends Supportive with EncryptSupportive {
         }
       }
 
+      val servableManager = new ServableManager
+      timing("load servable manager")() {
+        try servableManager.load(arguments.servableManagerPath)
+        catch {
+          case e: ServableLoadException =>
+            logger.info("ServableLoadException: " + e.message)
+            return
```

instead of logging, better to throw this exception